packages/bundler/src/modules/EventsManager.ts

```diff
@@ -34,8 +34,10 @@ export class EventsManager {
    * process all new events since last run
    */
   async handlePastEvents (): Promise<void> {
-    const fromBlock = this.lastBlock ?? Math.max(1, await this.entryPoint.provider.getBlockNumber() - 1000)
-    const events = await this.entryPoint.queryFilter({ address: this.entryPoint.address }, fromBlock)
+    if (this.lastBlock === undefined) {
```

Comment:
Note that this won't happen much - it can only happen if there are no past UserOperationEvent events.
Once there is a single such op (in the past 1000 blocks), the lastBlock is set (in line 57)
But I agree this pattern is cleaner way to set it.